src/ifcopenshell-python/ifcopenshell/file.py

```diff
@@ -464,6 +568,68 @@ def schema_version(self) -> tuple[int, int, int, int]:
             number = re.search(prefix + r"(\d)", schema)
             version.append(int(number.group(1)) if number else 0)
         return tuple(version)
+    
+    @property 
+    def mvd(self) -> str:
+        """
+        View Definition supported by the exporting application that is reflected in the IFC file
+        For example: 
+        “CoordinationView”
+        | “PresentationView”
+        | “StructuralAnalysisView”
+        | “FMHandOverView”
+        | “QuantityTakeOffAddOnView”
+        | “SpaceBoundary1stLevelAddOnView”
+        | “SpaceBoundary2ndLevelAddOnView”
+        """
+        return ','.join(self.parsed_description.mvd)
+    
+    @property 
+    def mvd_comments(self) -> str:
+        """
+        For example; 'Comment [This
+        export contains Boolean Operation geometry that may not be fully
+        supported by other application]'
+        
+        f.mvd_comments
+        'This export contains Boolean Operation geometry that may not be fully supported by other application'
+        """
+        return str(self.parsed_description.comments)
+    
+    @property
+    def mvd_exchange_requirements(self) -> str:
+        return str(self.parsed_description.exchangerequirement)
+    
+    @property
+    def mvd_options(self) -> str:
+        """
+        Option being used by the exporting application for e.g. quality control and debugging
+        a. this is an optional field providing informal information, the values for the Option keyword
+        reflect the export settings of the exporting software application, those settings may be
+        specific to the exporting software
+        For example: 
+        > "{'ExcludedObjects': [' Stair', ' Ramp', ' Space'], 'SplitLevel': ' On'}"
+        """
+        return str(self.parsed_description.option)
+    
+    @property 
+    def mvd_keywords(self) -> str:
+        """
+        Returns all keywords from description, e.g. 
+        f.mvd_keywords
+        "{'exchangerequirement', 'mvd', 'option', 'comment', 'remark'}"_
+        """
+        return str(self.parsed_description.keywords)
+    
+    @property
+    def mvd_optional_keyword_fields(self) -> str:
+        """"
+        Returns fields from keywords other than 'ViewDefinition', 'Option', 'Comment' and 'ExchangeRequirement'.
+        For example, in case of 
+        'REMARK [SomeKey: SomeValue; AnotherKey: AnotherValue]'
+        file.mvd_optional_keywords_fields == "{'remark': {'SomeKey': ' SomeValue', 'AnotherKey': ' AnotherValue'}}"
+        """
+        return str({kw: getattr(self.parsed_description, kw) for kw in self.parsed_description.other_keywords})
```

Comment:
I think I wouldn't make the distinction between `other_keywords` and the 4 ones you mentioned. Just list them all under the dictionary. Saves some logic to remove like keeping track of the set of keywords. Maybe build the dictionary at parse time and not here in the property getter